staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -476,7 +476,7 @@ func (qs *queueSet) chooseQueueIndexLocked(hashValue uint64, descr1, descr2 inte
 	// the dealer uses the current desired number of queues, which is no larger than the number in `qs.queues`.
 	qs.dealer.Deal(hashValue, func(queueIdx int) {
 		// TODO: Consider taking into account `additional latency` of requests
-		// in addition to their widths.
+		// in addition to their seats.
```

Comment:
This change is not necessary, the reason this PR is here is because width means seats.
This change is not a problem.